ark/analysis/visualize.py

```diff
@@ -63,14 +62,23 @@ def draw_boxplot(cell_data, col_name, col_split=None, split_vals=None, save_dir=
         misc_utils.save_figure(save_dir, "boxplot_viz.png")
 
 
-def visualize_z_scores(z, pheno_titles, save_dir=None):
+def visualize_z_scores(z, x_labels, y_labels, dpi=None, center_val=None,
+                       colormap="vlag", save_dir=None):
```

Comment:
One more thing I forgot: let's call this something different, since this function will be used for general heatmap plotting. Something like: `draw_heatmap`.